other/install_3-0_MySQL.sql

```diff
@@ -991,6 +991,17 @@ CREATE TABLE {$db_prefix}qanda (
 	INDEX idx_lngfile (lngfile)
 ) ENGINE={$engine};
 
+#
+# Table structure for table `reactions`
+#
+
+CREATE TABLE {$db_prefix}reactions (
+	id_reaction SMALLINT UNSIGNED AUTO_INCREMENT,
+	name VARCHAR(255) NOT NULL DEFAULT '',
+	order SMALLINT UNSIGNED NOT NULL DEFAULT '0',
```

Comment:
```suggestion
	`order` SMALLINT UNSIGNED NOT NULL DEFAULT '0',
```

Reply:
I dropped that column entirely for the time being. I was considering adding functionality to choose the order of the reactions but forgot about that when I wrote all the admin code and didn't want to go to the trouble of adding that in as well. I'll do that if I add it back at some point though.